jac-uk/apply
# Feature/773 update character check consent form

## What's included?
Candidates are able to log into their accounts when invited to do so via email and complete/view completed consent forms.

## Who should test?
✅ Product owner
✅ Developers
✅ UTG

## How to test?
(Please test together with [#1312](https://github.com/jac-uk/digital-platform/pull/570) and [#1327](https://github.com/jac-uk/admin/pull/1357)) [Here is the video description](https://drive.google.com/file/d/1BGKzj_PvCIjhXHCVM4M6KM3DlR29_zvI/view?usp=sharing) of expected functionality

- Apply for an exercise.
- When receive character checks request/reminder email, log into the Apply account, complete and submit consent form.
- Review submitted information.

## Risk - how likely is this to impact other areas?
🟠 Medium risk - this does change code that is shared with other areas

## Additional context
![Apply4](https://user-images.githubusercontent.com/40855898/125977383-0ea65ef2-d052-457b-9b63-9e4d31dc0c93.png)
![Apply1](https://user-images.githubusercontent.com/40855898/125976863-f860f591-7a62-4434-bf88-06583c64fdc2.png)
![Apply2](https://user-images.githubusercontent.com/40855898/125976866-975e44e8-5969-4dbf-a91b-7e9f4cf7bb2c.png)
![Apply3](https://user-images.githubusercontent.com/40855898/125976873-4beea7fd-3dd8-486b-8eb0-3c1ae424faff.png)


---
PREVIEW:DEVELOP
_can be OFF, DEVELOP or STAGING_
